ngraph/core/src/preprocess/pre_post_process.cpp

```diff
@@ -339,6 +392,20 @@ InputNetworkInfo&& InputNetworkInfo::set_layout(const Layout& layout) && {
     return std::move(*this);
 }
 
+InputTensorInfo& InputTensorInfo::set_color_format(const ov::preprocess::ColorFormat& format,
+                                                   const std::vector<std::string>& sub_names) & {
+    m_impl->set_planes_sub_names(sub_names);
+    m_impl->set_color_format(format);  // noexcept
+    return *this;
+}
+
+InputTensorInfo&& InputTensorInfo::set_color_format(const ov::preprocess::ColorFormat& format,
+                                                    const std::vector<std::string>& sub_names) && {
+    m_impl->set_planes_sub_names(sub_names);
```

Comment:
can we add a check that if we already have tensor with name `"name/" + sub_names[..]` we throw an exception that we have conflicts with tensor names?

Reply:
It can be quite long operation and looks quite basic check for any transformation. Maybe it would be better to add such check in 'validate_and_infer_types' at the end of preprocessing? @ilyachur what is your opinion?

Reply:
Added appropriate checks. For sub-names which result to non-unique tensors it will throw exception. For internal operations - it will search for available name